notebooks/236-raft-optimization/236-raft-optimization.ipynb

```diff
@@ -0,0 +1,570 @@
+    "# Convert and Optimize RAFT with OpenVINO™\n",
+    "\n",
+    "The Recurrent All-Pairs Field Transforms (RAFT) [repository](https://github.com/princeton-vl/RAFT) is a new deep network architecture for optical flow. \n",
+    "\n",
+    "RAFT extracts perpixel features, builds multi-scale 4D correlation volumes for all pairs of pixels, and iteratively updates a flow field through a recurrent unit that performs lookups on the correlation volumes. RAFT achieves stateof-the-art performance.\n",
+    "\n",
+    "This tutorial demonstrates step-by-step instructions on how to run and optimize RAFT with OpenVINO based on the [ONNX-RAFT-Optical-Flow-Estimation](https://github.com/ibaiGorordo/ONNX-RAFT-Optical-Flow-Estimation/tree/main). \n",
+    "\n",
+    "The tutorial consists of the following steps:\n",
+    "- Download ONNX model\n",
+    "- Convert ONNX model to OpenVINO IR\n",
+    "- Prepare and run optimization pipeline\n",
+    "- Visualize the flow image\n",
+    "- Evaluate the performance."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Download the ONNX model from the [PINTO_model_zoo](https://github.com/PINTO0309/PINTO_model_zoo/tree/04f86a550e2ce1c5fb70cfafdfdeb568700f61c2/252_RAFT). \n",
+    "The original PyTorch models were converted to ONNX models with different input sizes and iteration times.\n",
+    "\n",
+    "Here we will convert the raft_kitti_iter20_480x640 model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "sys.path.append(\"../utils\")\n",
+    "import notebook_utils as utils\n",
+    "import tarfile\n",
+    "from pathlib import Path\n",
+    "\n",
+    "base_model_dir = Path(\"models\")\n",
+    "\n",
+    "model_name = \"iter20\"\n",
+    "\n",
+    "archive_name = \"resources.tar.gz\"\n",
+    "\n",
+    "model_url = f\"https://s3.ap-northeast-2.wasabisys.com/pinto-model-zoo/252_RAFT/{model_name}/{archive_name}\"\n",
+    "\n",
+    "downloaded_model_path = base_model_dir / Path(archive_name)\n",
+    "\n",
+    "if not downloaded_model_path.exists():\n",
+    "    utils.download_file(model_url, downloaded_model_path.name, downloaded_model_path.parent)\n",
+    "\n",
+    "model_onnx_path= base_model_dir / Path('raft_kitti_iter20_480x640.onnx')\n",
+    "\n",
+    "if not model_onnx_path.exists():\n",
+    "    with tarfile.open(downloaded_model_path) as file:\n",
+    "        file.extractall(base_model_dir)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Convert model to OpenVINO IR."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "OpenVINO IR already converted\n"
+     ]
+    }
+   ],
+   "source": [
+    "import subprocess\n",
```

Comment:
jupyter notebook support native shell commands execution using 
! - symbol before command.
But could you please use mo python API for convert model instead of script? 
https://docs.openvino.ai/latest/openvino_docs_MO_DG_Python_API.html

```
ov_model = mo.convert_model(model_onnx_path)
```
also it will be good to use fp16 precision, using `compress_to_fp16=True` flag to reduce model size and get advantages of running on GPU